website/script.js

```diff
@@ -21,7 +22,12 @@ function getData(parameter) {
   xhttp.onreadystatechange = function() {
     if (this.readyState == 4 && this.status == 200) {
       var x = document.getElementById(parameter);
-      x.value = this.responseText;
+      var result = this.responseText;
+      x.value = result;
+      if (parameter == "callsign" && result != "") {
+        var y = document.getElementById("callsigntag");
+        y.innerHTML = " :: " + result;
+      }
```

Comment:
Would be nice to have this in a separate function (i.e. if parameter == "callsign" setCallsignTag(result);)

Reply:
[x] done. Will be included in the PR against the dev branch.